app/controllers/course/assessment/submission_question/comments_controller.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+class Course::Assessment::SubmissionQuestion::CommentsController < Course::Assessment::Controller
```

Comment:
The plan in future is to rename this to `Answer`, and the current `Answer` becomes `Attempt`.